packages/eslint-plugin/src/rules/space-infix-ops.ts

```diff
@@ -69,17 +69,16 @@ export default util.createRule<Options, MessageIds>({
 
     function isSpaceChar(token: TSESTree.Token): boolean {
       return (
-        token.type === AST_TOKEN_TYPES.Punctuator &&
-        /^[=|?|:]$/.test(token.value)
```

Comment:
this regexp was not correct, it should be, `/^[=?:]$/` or `/^(=|?|:)$/`

Reply:
I recommend [this plugin](https://ota-meshi.github.io/eslint-plugin-regexp/playground/#eJyrVkrOT0lVslLSj4u2rbGvsYpV0dcrSS0u0SjJz07N0ytLzClN1YzJU6oFACNGDhE=), if you have a lot of non-trivial regexps to lint